packages/frontend/src/pages/CreateEventPage/CreateEventPage.tsx

```diff
@@ -48,6 +52,29 @@ export const CreateEventPage = () => {
 
     setIsCreating(() => true);
 
+    const newEvent: IEvent = {
+      id: uuidv4(),
+      name: eventName,
+      description: eventDescription,
+      place: eventPlace,
+      startDate: eventStartDate,
+      startTime: eventStartTime,
+      endTime: eventEndTime,
+      duration: eventDuration,
+      registrationDate: eventRegistrationDate,
+      gender: gender as Gender,
+      ageMin: minVolunteersAge,
+      ageMax: maxVolunteersAge,
+      language: eventLanguage,
+      volunteersQuantity: volunteersQuantity,
+      status: eventStatus,
+      image: image as File,
+      endDate: eventEndDate,
+      telegramChannelLink,
+    };
+
+    console.log(newEvent);
```

Comment:
Maybe you forgot to delete console.log)

Reply:
No, I'm not. I will delete it on my next PR